### PR TITLE
Describe mergeColumnFieldValues and mergeRowFieldValues properties

### PR DESCRIPTION
**Pull request by AlexJameson**
image
image

### CODE REVIEW

**Review by RomanTsukanov — CHANGES_REQUESTED** [review]
It would be great to add illustrations that show what the exported file looks like with and without these settings.

File: api-reference/50 Common/Object Structures/ExcelExportPivotGridProps/mergeColumnFieldValues.md [hunk 1]
```diff
@@ -5,7 +5,13 @@ default: true
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Specifies whether to merge neighbouring cells in the column field if they have the same values.
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+The following image shows the exported pivot grid if you set this property to **true** or leave it unspecified...
```
Comment (RomanTsukanov) [inline api-reference/50 Common/Object Structures/ExcelExportPivotGridProps/mergeColumnFieldValues.md]:
```suggestion
The following image shows the exported pivot grid if you set this property to **true** or leave it unspecified:
```

File: api-reference/50 Common/Object Structures/ExcelExportPivotGridProps/mergeColumnFieldValues.md [hunk 2]
```diff
@@ -5,7 +5,13 @@ default: true
...
+
+![DevExtreme PivotGrid: mergeColumnFieldValues set to true](/images/UiWidgets/pivotgrid-merge-column-field-values-true.png)
+
+... and in this image this property is set to false:
```
Comment (RomanTsukanov) [inline api-reference/50 Common/Object Structures/ExcelExportPivotGridProps/mergeColumnFieldValues.md]:
```suggestion
In the image below, the **mergeColumnFieldValues** property is set to **false**:
```

File: api-reference/50 Common/Object Structures/ExcelExportPivotGridProps/mergeRowFieldValues.md [hunk 3]
```diff
@@ -5,7 +5,13 @@ default: true
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Specifies whether to merge neighbouring cells in the row field if they have the same values.
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+The following image shows exported pivot grid if you set this property to **true** or leave it unspecified...
```
Comment (RomanTsukanov) [inline api-reference/50 Common/Object Structures/ExcelExportPivotGridProps/mergeRowFieldValues.md]:
```suggestion
The following image shows exported pivot grid if you set this property to **true** or leave it unspecified:
```

File: api-reference/50 Common/Object Structures/ExcelExportPivotGridProps/mergeRowFieldValues.md [hunk 4]
```diff
@@ -5,7 +5,13 @@ default: true
...
+
+![DevExtreme PivotGrid: mergeRowFieldValues set to true](/images/UiWidgets/pivotgrid-merge-row-field-values-true.png)
+
+... and in this image this property is set to false:
```
Comment (RomanTsukanov) [inline api-reference/50 Common/Object Structures/ExcelExportPivotGridProps/mergeRowFieldValues.md]:
```suggestion
In the image below, the **mergeRowFieldValues** property is set to **false**:
```

**Review by RomanTsukanov — CHANGES_REQUESTED** [review]

File: api-reference/50 Common/Object Structures/ExcelExportPivotGridProps/mergeColumnFieldValues.md [hunk 5]
```diff
@@ -5,7 +5,13 @@ default: true
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Specifies whether to merge neighbouring cells in the column field if they have the same values.
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+The following image shows the exported pivot grid if you set this property to **true** or leave it unspecified:
```
Comment (dirk-pieterse) [inline api-reference/50 Common/Object Structures/ExcelExportPivotGridProps/mergeColumnFieldValues.md:11]:
```suggestion
The following images show the exported pivot grid if you set this property to **true** or leave it unspecified
```

File: api-reference/50 Common/Object Structures/ExcelExportPivotGridProps/mergeColumnFieldValues.md [hunk 6]
```diff
@@ -5,7 +5,13 @@ default: true
...
+
+![DevExtreme PivotGrid: mergeColumnFieldValues set to true](/images/UiWidgets/pivotgrid-merge-column-field-values-true.png)
+
+In the image below, the **mergeColumnFieldValues** property is set to **false**:
```
Comment (dirk-pieterse) [inline api-reference/50 Common/Object Structures/ExcelExportPivotGridProps/mergeColumnFieldValues.md:15]:
```suggestion
or set the **mergeColumnFieldValues** property to **false**:
```

File: api-reference/50 Common/Object Structures/ExcelExportPivotGridProps/mergeRowFieldValues.md [hunk 7]
```diff
@@ -5,7 +5,13 @@ default: true
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Specifies whether to merge neighbouring cells in the row field if they have the same values.
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+The following image shows exported pivot grid if you set this property to **true** or leave it unspecified:
```
Comment (dirk-pieterse) [inline api-reference/50 Common/Object Structures/ExcelExportPivotGridProps/mergeRowFieldValues.md:11]:
```suggestion
The following images show the exported pivot grid if you set this property to **true** or leave it unspecified
```

File: api-reference/50 Common/Object Structures/ExcelExportPivotGridProps/mergeRowFieldValues.md [hunk 8]
```diff
@@ -5,7 +5,13 @@ default: true
...
+
+![DevExtreme PivotGrid: mergeRowFieldValues set to true](/images/UiWidgets/pivotgrid-merge-row-field-values-true.png)
+
+In the image below, the **mergeRowFieldValues** property is set to **false**:
```
Comment (dirk-pieterse) [inline api-reference/50 Common/Object Structures/ExcelExportPivotGridProps/mergeRowFieldValues.md:15]:
```suggestion
or set the **mergeRowFieldValues** property to **false**:
```

**Review by RomanTsukanov — APPROVED** [review]

**Review by dirk-pieterse — APPROVED** [review]

File: api-reference/50 Common/Object Structures/ExcelExportPivotGridProps/mergeColumnFieldValues.md (hunk 5 above)
Comment (AlexJameson) [reply api-reference/50 Common/Object Structures/ExcelExportPivotGridProps/mergeColumnFieldValues.md:11]: Hello, @dirk-pieterse. Tech writers recently had a discussion and we decided not to divide a single sentence with images (or code blocks). It is better to create an unordered list or use some kind of the following text: 
To do A, assign B to the property: (image)
If you need C, set the property to D.

Also I'll write a message in English for you and Armina to sum up all our thoughts on this topic.

Comment (dirk-pieterse) [reply api-reference/50 Common/Object Structures/ExcelExportPivotGridProps/mergeColumnFieldValues.md:11]: Ok

**Pull request merged**